lua/pac3/extra/server/contraption.lua

```diff
@@ -42,6 +42,9 @@ local function spawn(val,ply)
 
 	if phys:IsValid() then
 		phys:EnableMotion(false)
+		for i=0, ent:GetBoneCount() do
```

Comment:
Should be `ent:GetBoneCount()-1`, also why is the vector getting the x,y,z components out of order?

Reply:
> Should be `ent:GetBoneCount()-1`, also why is the vector getting the x,y,z components out of order?

If you do that won't that make the last bone not be set?
IDK why they are out of order but re-ordering them that way works if you look at `client/contraption` I got the size and scale directly. It may have something to do with the way pac3 scale and applied to to models

Reply:
The last bone is index `ent:GetBoneCount()-1` because they are 0 indexed. I'll need to look at why vector components are they way.